pyartcd/pyartcd/pipelines/ocp4_scan.py

```diff
@@ -54,7 +54,10 @@ async def run(self):
 
             # Update build description
             if image_list:
-                jenkins.update_description(f'Changed images: {",".join(image_list)}<br/>')
+                if len(image_list) > 100:
```

Comment:
Not sure if `100` is still too high as a threshold, besides the description would not be very readable anyway. I'd consider lowering it down